<?xml version="1.0" ?>

<launch>    
    
	<!-- Check USER ID -->
    	
    <arg name="host_name"          		 		 value="$(env ALICE_HOST)"/>
	
    <arg name="id1_truth"             		 value="$(eval host_name=='alice1nuke')"/>
    <arg name="id2_truth"             		 value="$(eval host_name=='alice2nuke')"/>
    <arg name="alice_uid" value="1" unless="$(arg id2_truth)" /> 
    <arg name="alice_uid" value="2" unless="$(arg id1_truth)" /> 

    	
   <!-- <arg name="alice_uid" value="1" /> -->
    
    <param name="alice_userid"             	 value="$(arg alice_uid)"/>
 
    <!-- GAZEBO Robot Setting -->	
    	
    <param name="gazebo"                 	 value="false"      type="bool"/>
    
    <arg name="robot_id1_truth"             		 value="$(eval alice_uid==1)"/>
    <arg name="robot_id2_truth"             		 value="$(eval alice_uid==2)"/> 	
    <arg name="robot_name" 	value="alice_2_robot" 	unless="$(arg robot_id2_truth)" />
    <arg name="robot_name" 	value="alice_robot" 	unless="$(arg robot_id1_truth)" />
      
    <param name="gazebo_robot_name"        	 value="$(arg robot_name)" />
   	    	
    <!-- YAML FILE LOAD -->
    <param name="offset_file_path"           value="$(find alice_manager)/config/offset_$(arg alice_uid).yaml"/>
    <param name="robot_file_path"            value="$(find alice_manager)/config/alice2_$(arg alice_uid).robot"/>
    <param name="init_file_path"             value="$(find alice_manager)/config/dxl_init_$(arg alice_uid).yaml"/>
   		   
    <node name="offset_tuner_server" pkg="offset_tuner_server" type="offset_tuner_server" output="screen">
    </node>
    
</launch>
